com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceAmbientOcclusion.cs

```diff
@@ -178,7 +178,10 @@ internal bool Setup(ScreenSpaceAmbientOcclusionSettings featureSettings, Scripta
                 }
                 else
                 {
-                    renderPassEvent = featureSettings.AfterOpaque ? RenderPassEvent.AfterRenderingOpaques : RenderPassEvent.AfterRenderingPrePasses;
+                    // Rendering after PrePasses is usually correct except when depth priming is in play:
+                    // then we rely on a depth resolve taking place after the PrePasses in order to have it ready for SSAO.
+                    // Hence we set the event to BeforeRenderingOpaques at the earliest.
+                    renderPassEvent = featureSettings.AfterOpaque ? RenderPassEvent.AfterRenderingOpaques : RenderPassEvent.BeforeRenderingOpaques;
```

Comment:
I'm worried we might introduce a regression if there's a custom pass that injects also in BeforeRenderingOpaques but run before SSAO. 
 
 Safer approach can be to do do `RenderPassEvent.AfterRenderingPrePasses + 1` (this is just a prioriity queue).

Reply:
Fixed